running-requests/prompt-blueprints.mdx

```diff
@@ -132,3 +132,48 @@ response = pl_client.run(
 
 print(response)
 ```
+
+
+## Using Image Variable
+
+The `image_variable` attribute allows you to dynamically insert a list of images into your prompt templates. This attribute is part of the `ImageContent` structure and can be used to include images based on a specified variable name.
```

Comment:
What is `ImageContent`?

Reply:
We have two types of `content` inside a message body
1. `TextContent` which is supposed to contain text
2. `ImageContent` which is supposed to contain image content